storage/container_storage_interface/persistent-storage-csi-ebs.adoc

```diff
@@ -0,0 +1,30 @@
+[id="persistent-storage-csi-ebs"]
+= AWS Elastic Block Store CSI Operator
+include::modules/common-attributes.adoc[]
+:context: persistent-storage-csi-ebs
+
+toc::[]
+
+== Overview
+
+Storage vendors have traditionally provided storage drivers as part of Kubernetes. With the implementation of the Container Storage Interface (CSI), third-party providers can instead deliver storage plug-ins using a standard interface without ever having to change the core Kubernetes code.
+
+CSI Operators give {product-title} users storage options that are not possible with in-tree volume plug-ins, such as volume snapshots and cloning.
+
+[IMPORTANT]
+====
+While {product-title} also supports in-tree storage for AWS EBS and other third-party providers, Kubernetes will be removing in-tree drivers in the future.
```

Comment:
It might be worthwhile mentioning that volumes using the in-tree plugin will be migrated to use the CSI driver.

Reply:
> It might be worthwhile mentioning that volumes using the in-tree plugin will be migrated to use the CSI driver.

@bertinatto By "will be migrated", do you mean in the future when in-tree is dropped? Assuming that's the case, I have revised to the following:

- > While {product-title} also supports in-tree storage for AWS EBS and other third-party providers, Kubernetes will be removing in-tree drivers in the future. *Volumes using an in-tree plug-in will be migrated to use of the CSI driver at that time.*

But if they will automatically be migrated with implementation of this CSI AWS EBS Operator (4.5), then I would need to state it differently.

Reply:
Got clarification from Christian on this and updated accordingly: 

> {product-title} defaults to using an in-tree, or non-CSI, driver to provision AWS EBS storage. This in-tree driver will be removed in a subsequent update of {product-title}. Volumes provisioned using the existing in-tree driver are planned to be migrated to the CSI driver at that time.